src/builder-context.c

```diff
@@ -344,6 +346,32 @@ builder_context_find_in_sources_dirs_va (BuilderContext *self,
   return NULL;
 }
 
+void 
+builder_context_set_opt_export_only (BuilderContext *self,
+                                          gboolean       opt_export_only)
```

Comment:
Some weird things going on with this second argument indentation & `builder_context_set_opt_mirror_screenshots_url`...